manager/controllers/utils/configuration.go

```diff
@@ -16,6 +16,7 @@ import (
 // Attributes that are defined in a config map or the runtime environment
 const (
 	CatalogConnectorServiceAddressKey string = "CATALOG_CONNECTOR_URL"
+	VaultEnabled                      string = "VAULT_ENABLED"
```

Comment:
All other const have `key` suffix...

Reply:
what do you mean?

Reply:
`CatalogConnectorServiceAddressKey`, `VaultAddressKey` ...